examples/connext_dds/fragmented_data_statistics/c++/application.h

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
Don't think this file should be committed

Reply:
It is needed to allow compile the examples.